libs/datamodel/core/src/ast/parser/parse_attribute.rs

```diff
@@ -44,7 +44,7 @@ fn parse_attribute_args(token: &Token<'_>, arguments: &mut Vec<Argument>) {
     }
 }
 
-fn parse_attribute_arg(token: &Token<'_>) -> Argument {
+pub fn parse_attribute_arg(token: &Token<'_>) -> Argument {
```

Comment:
Why did this need to become public?

Reply:
It didn't, I made it pub(crate) now since I only need it in parse_expression now